### PR TITLE
Added documentation for Bitset

### PR DESCRIPTION
**Pull request by nliber**
(with @ldh4)

Added documentation for Bitset
Fixed links for vector

### CODE REVIEW

File: API-Reference/API-Alphabetical.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ All functions and classes listed here are part of the `Kokkos::` namespace.
 ### Containers
 |Name |Library | Category | Description                  |
 |:---------|:--------|:-----------|:----------------------------|
-|[Bitset](Kokkos%3A%3ABitset) | [Containers](API-Containers) | View | A concurrent Bitset class. |
+|[Bitset](Bitset) | [Containers](API-Containers) | View | A concurrent Bitset class. |
```
Comment (dalg24) [inline API-Reference/API-Alphabetical.md:19]: Did you forget to commit that file?
Also why did you not name it `[Kokkos::]Bitset` for consistency?

File: API-Reference/API-Containers.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
-### 1. **[[Bitset|bitset-path]]**
+### 1. **[[Bitset|Bitset]]**
 Bitset comment
```
Comment (dalg24) [inline API-Reference/API-Containers.md:2]: This would need to be updated too

File: API-Reference/API-Alphabetical.md (hunk 1 above)
Comment (nliber) [reply API-Reference/API-Alphabetical.md:19]: The problem is we really aren't consistent.  We saw Unordered-Map.md in the top level directory, and started with that as the template for writing Bitset.

I don't mind changing it or moving the file.  Should I?

Comment (crtrott) [reply API-Reference/API-Alphabetical.md:19]: Yeah change it back. Note Unorderedmap is not up to snuff in how it is written/structured.

File: _Sidebar.md [hunk 3]
```diff
@@ -15,11 +15,13 @@
 12. **[[Interoperability|Interoperability]]**
 13. **[[Kokkos and Virtual Functions|Kokkos and Virtual Functions]]**
 ### **[[Content: Containers|ContentContainers]]**
-1. **[DualView](Kokkos%3A%3ADualView)**
-2. **[DynRankView](Kokkos%3A%3ADynRankView)**
-3. **[DynamicView](Kokkos%3A%3ADynamicView)**
-4. **[[OffsetView|Offset-View]]**
-5. **[[UnorderedMap|Unordered-Map]]**
+1. **[[Bitset|Bitset]]**
```
Comment (crtrott) [inline _Sidebar.md:18]: Note this is the programming guide link. This should not ahve the links to the API Reference. I think the links for most of the other classes here are wrong too.

**Review by crtrott — CHANGES_REQUESTED** [review]

File: _Sidebar.md (hunk 3 above)
Comment (nliber) [reply _Sidebar.md:18]: I'll move all the containers from programming guide section of the sidebar to the API reference section.

**Review by crtrott — CHANGES_REQUESTED** [review]
I think we need to improve this and bring this more in line how View documentation is structured. I don't want documentation as comment in a synopsis. I know some of the other containers were documented like that, but we need to fix those, not add more classes in that style.

**Comment by nliber** [comment]
Reformatted `Bitset` in the style of view as per @crtrott comment above.

**Review by crtrott — APPROVED** [review]

**Review by dalg24 — APPROVED** [review]

**Pull request merged**